week8/homepage/Demon_Slayer.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+
+<html lang="en">
+
+<head>
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" rel="stylesheet"
+        integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC" crossorigin="anonymous">
+    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/js/bootstrap.bundle.min.js"
+        integrity="sha384-MrcW6ZMFYlzcLA8Nl+NtUVF0sA7MsXsP1UyJoMp4YLEuNSfAP+JcXn/tWtIaxVXM"
+        crossorigin="anonymous"></script>
+    <link href="styles.css" rel="stylesheet">
+    <title>Demon Slayer</title>
+</head>
+
+<body class="body">
+    <div class="header">
+        <h1>Demon Slayer</h1>
+    </div>
+
+    <div class="topnav">
+        <a href="file:///Users/marcrader/CS50Kurs/week8/homepage/index.html">Homepage</a>
+        <a href="file:///Users/marcrader/CS50Kurs/week8/homepage/OnePiece.html">One Piece</a>
+        <a href="file:///Users/marcrader/CS50Kurs/week8/homepage/Naruto.html">Naruto</a>
+        <a href="file:///Users/marcrader/CS50Kurs/week8/homepage/Demon_Slayer.html">Demon Slayer</a>
+        <a href="file:///Users/marcrader/CS50Kurs/week8/homepage/Dragonball_Z.html">Dragonball Z</a>
+        <a href="file:///Users/marcrader/CS50Kurs/week8/homepage/Userfavourite.html">Your favourites</a>
```

Comment:
Use relative paths instead of absolute paths.

![Screenshot 2022-09-02 at 09 35 14](https://user-images.githubusercontent.com/80166473/188083895-a2920200-9566-48fa-9ddc-f1eeeb6a1b7f.png)